aggregator.go

```diff
@@ -162,6 +165,9 @@ func (r *reverseMinNorm) Normalize(w float64) float64 {
 }
 
 func (r *sigmoidNorm) Normalize(w float64) float64 {
+	if r.scale == 0 {
```

Comment:
r.scale == 0 || w == 0

Reply:
I don't know if it is needed here, because `scale` is a fixed parameter, not some data-dependent value.

Reply:
@fyrchik I guess it could be checked outside then:

```go
func getDefaultWeightFunc(ns Nodes) WeightFunc {
	agg := new(meanCapAgg)
	for i := range ns {
		agg.Add(ns[i])
	}
        // check for agg.Compute() == 0 here
	return NewWeightFunc(&sigmoidNorm{agg.Compute()}, &constNorm{1})
}
```